smoke_test.py

```diff
@@ -1,5 +1,5 @@
 import sys
-full = sys.argv[1] == 'full'
```

Comment:
would be nicer to use an arg parser imo

Reply:
This case (just a single positional argument) is simple enough.